be/src/storage/lake/tablet_manager.cpp

```diff
@@ -239,11 +244,17 @@ StatusOr<TabletMetadataPtr> TabletManager::get_tablet_metadata(int64_t tablet_id
 }
 
 StatusOr<TabletMetadataPtr> TabletManager::get_tablet_metadata(const string& path, bool fill_cache) {
+    std::shared_ptr<FileSystem> fs;
+    return get_tablet_metadata(fs, path, fill_cache);
```

Comment:
why pass a invalid fs pointer as a parameter?

Reply:
remove it